env_vars.tf

```diff
@@ -44,7 +44,8 @@ data "ns_secret_keys" "this" {
 }
 
 locals {
-  secret_keys  = data.ns_secret_keys.this.secret_keys
-  all_secrets  = data.ns_env_variables.this.secrets
-  all_env_vars = merge(data.ns_env_variables.this.env_variables, local.app_secret_ids)
+  secret_keys          = data.ns_secret_keys.this.secret_keys
+  all_secrets          = data.ns_env_variables.this.secrets
+  existing_secret_refs = data.ns_env_variables.this.secret_refs
+  all_env_vars         = merge(data.ns_env_variables.this.env_variables, local.app_secret_ids, local.existing_secret_refs)

```

Comment:
This seems like it would produce an inconsistent result for the developer.

`local.app_secret_ids` will look like `<env_var>_SECRET_ID=<secret-arn>`
`local.existing_secret_refs` will look like `<env_var>=<secret_arn>`